venthur_api/main.py

```diff
@@ -100,18 +96,24 @@ def server_response(string, connection):
 
 def parse_command(command):
```

Comment:
from a semantic standpoint, this should be renamed to `get_method` or words to those effect